README.rst

```diff
@@ -58,13 +58,12 @@ You can run only the boto3 tests with::
  STS compatibility tests
 ========================
 
-This section contains some basic tests for the AssumeRole and GetSessionToken API's. The test file is located under ``s3tests_boto3/functional``.
+This section contains some basic tests for the AssumeRole, GetSessionToken and AssumeRoleWithWebIdentity API's. The test file is located under ``s3tests_boto3/functional``.
 
-You can run only the sts tests with::
+You can run only the sts tests (all the three API's) with::
 
         S3TEST_CONF=your.conf ./virtualenv/bin/nosetests s3tests_boto3.functional.test_sts
 
-You can filter tests based on the attributes. There is a attribute named ``sts_test`` to run specifically the sts tests as below::
-
-        S3TEST_CONF=your.conf ./virtualenv/bin/nosetests -v -s -A 'sts_test' s3tests_boto3.functional.test_sts
+You can filter tests based on the attributes. There is a attribute named ``assume_role_test``, ``get_session_token_test`` and ``webidentity_test`` to run specifically the tests associated with the specific API's as below::
 
+        S3TEST_CONF=your.conf ./virtualenv/bin/nosetests -v -s -A 'name_of_the_API_test' s3tests_boto3.functional.test_sts
```

Comment:
Minor nit, just replace `name_of_the_API_test` with `assume_role_test,get_session_token_test,webidentity_test` as an example for running all of them.